metricbeat/docs/modules/zookeeper.asciidoc

```diff
@@ -3,20 +3,22 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[metricbeat-module-zookeeper]]
-== Zookeeper Module
+== ZooKeeper Module
 
+//REVEWIERS: We need to add a description here. What would you suggest? 
```

Comment:
The ZooKeeper modules fetches statistic from the ZooKeeper service (link to service...).